pkg/common/kafka/events_test.go

```diff
@@ -0,0 +1,81 @@
+package kafkacommon_test
+
+import (
+	"time"
+
+	"github.com/bxcodec/faker/v3"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhatinsights/edge-api/config"
+	kafkacommon "github.com/redhatinsights/edge-api/pkg/common/kafka"
+	"github.com/redhatinsights/edge-api/pkg/models"
+	"github.com/redhatinsights/edge-api/pkg/routes/common"
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+)
+
+var _ = Describe("Test Create Edge Event", func() {
+	var ctrl *gomock.Controller
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		config.Init()
+
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+	Describe("Get Consumer", func() {
+		When("One does not exist yet", func() {
+			Context("One does not exist yet", func() {
+				It("should be ok", func() {
+					var ident identity.XRHID
+					ident.Identity.OrgID = common.DefaultOrgID
+					imageSet := &models.ImageSet{
+						Name:    "test",
+						Version: 2,
+						OrgID:   common.DefaultOrgID,
+					}
+					image := &models.Image{
+						Commit: &models.Commit{
+							OSTreeCommit: faker.UUIDHyphenated(),
+							OrgID:        common.DefaultOrgID,
+							Status:       "BUILDING",
+						},
+						Status:     models.ImageStatusSuccess,
+						ImageSetID: &imageSet.ID,
+						Version:    1,
+						OrgID:      common.DefaultOrgID,
+						Name:       "test",
+					}
+					edgePayload := &models.EdgeImageRequestedEventPayload{
+						EdgeBasePayload: models.EdgeBasePayload{
+							Identity:       ident,
+							LastHandleTime: time.Now().Format(time.RFC3339),
+							RequestID:      uuid.New().String(),
+						},
+						NewImage: *image,
+					}
+
+					cloudEvent := models.CRCCloudEvent{
+						Data:        edgePayload,
+						DataSchema:  "v1",
+						ID:          faker.HyphenatedID,
+						RedHatOrgID: common.DefaultOrgID,
+						Source:      models.SourceEdgeEventAPI,
+						SpecVersion: "v1",
+						Subject:     image.Name,
+						Time:        time.Now().Format(time.RFC3339),
+						Type:        models.EventTypeEdgeImageRequested,
+					}
+					event := kafkacommon.CreateEdgeEvent(common.DefaultOrgID, models.SourceEdgeEventAPI, image.RequestID, models.EventTypeEdgeImageRequested, image.Name, edgePayload)
+					Expect(event.Data).To(Equal(cloudEvent.Data))
+					Expect(event.RedHatOrgID).To(Equal(cloudEvent.RedHatOrgID))
```

Comment:
maybe like this: 
```suggestion
					Expect(event.Data).To(Equal(edgePayload))
					Expect(event.RedHatOrgID).To(Equal(common.DefaultOrgID))
```

and remove cloudEvent variable, 

what do you think ?